docs/sources/write/style-guide/word-list/index.md

```diff
@@ -34,17 +55,21 @@ Use this rather than _datasource_ for the noun form.
 
 Also, use _data source plugin_ rather than _data-source plugin_.
 
-While most compound adjectives require a hyphen, we've chosen to leave it out in this case to maintain consistency with the naming of data sources in the application and reduce confusion.
+While most compound adjectives require a hyphen, it's left out in this case to maintain consistency with the naming of data sources in the application and reduce confusion.
 
 {{< admonition type="note" >}}
 For other compound adjectives, use a hyphen unless otherwise specified.
 {{< /admonition >}}
 
 ### dialog box
 
+<!-- vale Grafana.DialogBox = NO -->
+
 Use this rather than _modal_ or _dialog_.
 
-This guidance intentionally differs from Google style guide advice which prefers just [_dialog_](https://developers.google.com/style/word-list#dialog) because _dialog box_ is a user friendly term that is easy to understand.
+This guidance intentionally differs from Google style guide advice which prefers just [_dialog_](https://developers.google.com/style/word-list#dialog) because _dialog box_ is a user friendly term that's easy to understand.
```

Comment:
```suggestion
This guidance intentionally differs from Google style guide advice, which prefers just [_dialog_](https://developers.google.com/style/word-list#dialog) because _dialog box_ is a user-friendly term that's easy to understand.
```